packages/components/src/components/link-tile-model-signature/link-tile-model-signature.tsx

```diff
@@ -0,0 +1,118 @@
+import { Component, Element, h, JSX, Prop } from '@stencil/core';
+import { MODEL_SIGNATURE_MODELS } from '../model-signature/model-signature-utils';
+import type { BreakpointCustomizable, PropTypes } from '../../types';
+import { GROUP_DIRECTIONS } from '../../styles/group-direction-styles';
+import {
+  AllowedTypes,
+  attachComponentCss,
+  getNamedSlotOrThrow,
+  getPrefixedTagNames,
+  throwIfElementIsNotOfKind,
+  TILE_ASPECT_RATIOS,
+  TILE_WEIGHTS,
+  validateProps,
+} from '../../utils';
+import { getComponentCss } from './link-tile-model-signature-styles';
+import type {
+  LinkTileModelSignatureAspectRatio,
+  LinkTileModelSignatureHeadingTag,
+  LinkTileModelSignatureLinkDirection,
+  LinkTileModelSignatureModel,
+  LinkTileModelSignatureWeight,
+} from './link-tile-model-signature-utils';
+import {
+  LINK_TILE_MODEL_SIGNATURE_HEADING_TAGS,
+  setRequiredPropsOfSlottedLinks,
+} from './link-tile-model-signature-utils';
+
+const propTypes: PropTypes<typeof LinkTileModelSignature> = {
+  model: AllowedTypes.oneOf<LinkTileModelSignatureModel>(MODEL_SIGNATURE_MODELS),
+  weight: AllowedTypes.breakpoint<LinkTileModelSignatureWeight>(TILE_WEIGHTS),
+  aspectRatio: AllowedTypes.breakpoint<LinkTileModelSignatureAspectRatio>(TILE_ASPECT_RATIOS),
+  heading: AllowedTypes.string,
+  description: AllowedTypes.string,
+  linkDirection: AllowedTypes.breakpoint<LinkTileModelSignatureLinkDirection>(GROUP_DIRECTIONS),
+  headingTag: AllowedTypes.oneOf<LinkTileModelSignatureHeadingTag>(LINK_TILE_MODEL_SIGNATURE_HEADING_TAGS),
+};
+
+@Component({
+  tag: 'p-link-tile-model-signature',
+  shadow: true,
```

Comment:
Wouldn't focus delegation be helpful?

Reply:
Thought about it, but we have the two links that can get focused in the light dom. The component it self cant be focused?